pom.xml

```diff
@@ -83,6 +83,7 @@
     <version.com.icegreen>1.4.1</version.com.icegreen>
     <version.javaee.spec>7.0</version.javaee.spec>
     <version.maven-patch-plugin>1.2</version.maven-patch-plugin>
+    <version.org.cassalog>0.3.0</version.org.cassalog>
```

Comment:
@lucasponce, @jsanda has released an important fix in cassaloge 0.3.1, please update.